extensions/kubeapi/projects/update.go

```diff
@@ -0,0 +1,45 @@
+package projects
+
+import (
+	"context"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/unstructured"
+	"github.com/rancher/shepherd/pkg/api/scheme"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// UpdateProject is a helper function that uses the dynamic client to update a project in a specific cluster.
+func UpdateProject(client *rancher.Client, updatedProject *v3.Project) (*v3.Project, error) {
+	dynamicClient, err := client.GetDownStreamClusterClient(LocalCluster)
```

Comment:
Same as the above comment, should this be a local cluster?

Reply:
Project CRDs are available only on the local cluster and not on the downstream cluster.